### PR TITLE
Update resource-health-to-troubleshoot-connectivity.md

### PR DESCRIPTION
**Pull request by tnbarkhouse**
Replaced by internal PR https://github.com/MicrosoftDocs/sql-docs-pr/pull/29440.

I recently updated the Geneva Monitors for Resource Health Checks for Azure SQL Database. The proposed changes are more accurate for the current behavior of RHC alerts.

### CODE REVIEW

File: azure-sql/database/resource-health-to-troubleshoot-connectivity.md [hunk 1]
```diff
@@ -29,19 +29,23 @@ ms.custom: sqldbrb=2
...
 
-A status of **Available** means that **Resource health** has not detected login failures due to system errors on your SQL database.
+A status of **Available** means that **Resource health** has not detected login failures due to system errors on your SQL database, or that there were some login failures but they did not meet the alerting threshold. See the following sections for more details on the alerting threshold.
 
 :::image type="content" source="./media/resource-health-to-troubleshoot-connectivity/sql-resource-health-available.jpg" alt-text="A screenshot of the Azure portal showing the status message for the state of Available.":::
 
 ### Degraded
 
-A status of **Degraded** means that **Resource health** has detected a majority of successful logins, but some failures as well. These are most likely transient login errors. To reduce the impact of connection issues caused by transient login errors, implement [retry logic](troubleshoot-common-connectivity-issues.md#retry-logic-for-transient-errors) in your code.
+A status of **Degraded** means that, in two of the last three minutes, **Resource health** has detected:
```
Comment (dimitri-furman) [inline azure-sql/database/resource-health-to-troubleshoot-connectivity.md:38]: > in two of the last three minutes

Which two specifically? The first and the second, or the second and the third? Can you rephrase to remove the ambiguity?

File: azure-sql/database/resource-health-to-troubleshoot-connectivity.md [hunk 2]
```diff
@@ -29,19 +29,23 @@ ms.custom: sqldbrb=2
...
+- more than one login failure (due to system errors) but there were fewer than six total login attempts.
+
+These are most likely transient login errors. To reduce the impact of connection issues caused by transient login errors, implement [retry logic](troubleshoot-common-connectivity-issues.md#retry-logic-for-transient-errors) in your code.
 
 :::image type="content" source="./media/resource-health-to-troubleshoot-connectivity/sql-resource-health-degraded.jpg" alt-text="A screenshot of the Azure portal showing the status message for the state of Degraded.":::
 
 ### Unavailable
 
-A status of **Unavailable** means that **Resource health** has detected consistent login failures to your SQL database. If your resource remains in this state for an extended period of time, contact support.
+A status of **Unavailable** means that **Resource health** has detected that there were more than five login attempts and more than a quarter of them were failing for system reasons. If your resource remains in this state for an extended period of time, please contact support.
```
Comment (dimitri-furman) [inline azure-sql/database/resource-health-to-troubleshoot-connectivity.md:48]: > that there were more than five login attempts

Over which time range? If we mention that when talking about degraded, we should mention it when talking about unavailable too.

File: azure-sql/database/resource-health-to-troubleshoot-connectivity.md [hunk 3]
```diff
@@ -51,6 +55,10 @@ The health status of **Unknown** indicates that **Resource health** hasn't recei
 
 :::image type="content" source="./media/resource-health-to-troubleshoot-connectivity/sql-resource-health-unknown.jpg" alt-text="A screenshot of the Azure portal showing the status message for the state of Unknown.":::
 
+## Alert Time
+
+The time shown by the **Resource health** alert will not line up with the times of the login failures that caused the alert. This is because it takes several minutes for the telemetry to be collected and analyzed, to determine that there is a **Resource health** issue. So, the time indicated in the **Resource health** alert will be several minutes after the login failures.
```
Comment (dimitri-furman) [inline azure-sql/database/resource-health-to-troubleshoot-connectivity.md:60]: IIRC, this would be a relevant statement to add here. If so, can you add?

>Additionally, the time interval when the login failures were occurring might often be shorter than the time interval in the resource health alert.

**Review by dimitri-furman — CHANGES_REQUESTED** [review]

**Comment by rwestMSFT** [comment]
@tnbarkhouse We need to move this PR internally, since you're an employee.

**Comment by rwestMSFT** [comment]
Replaced by https://github.com/MicrosoftDocs/sql-docs-pr/pull/29440.
